### ISSUE TITLE
snapping breaks after removing zoning

### ISSUE THREAD

**Issue by DuffBee** [zeeshanabid94/ZoningToolkit#9]
the snapping of ploppables does not work anymore on roads where zoning has been removed. the building will snap onto some invisible barrier in some distance to the road. readding the zoning to the road fixes this issue.  
![image](https://github.com/user-attachments/assets/fe39f7a5-beda-4ec6-aef7-4d8c60e28318)

**Comment by zeeshanabid94** [comment]
Yup. As far as I can tell, the game snaps using zoning for all ploppables. Can look into improving this. 

However, have you tried the work around where you disable zoning after placing the ploppable?

**Comment by DuffBee** [comment]
Yeah that kinda works. In my experience its a bit flaky as zoning cant be removed anymore after something is placed or grown on that area.
